src/main/java/com/amazon/inspector/jenkins/amazoninspectorbuildstep/AmazonInspectorBuilder.java

```diff
@@ -468,7 +468,7 @@ public AmazonInspectorBuilder newInstance(StaplerRequest req, JSONObject formDat
                 }
             }
 
-            boolean thresholdEnabled = formData.optBoolean("isThresholdEnabled", true);
+            boolean thresholdEnabled = formData.optBoolean("isThresholdEnabled", false);
```

Comment:
Sorry, I have no idea how Jenkins works. Is this generating the page each time the code is run? Should this `isThresholdEnabled` value be read from the `formData` variable? That would prevent it from reverting each time the page is loaded.